api/Navigator.json

```diff
@@ -2226,8 +2226,8 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": false,
-            "deprecated": false
+            "standard_track": true,
```

Comment:
Confirmed: https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-vendorsub

(Must return the empty string.)